### PR TITLE
feat(webapp): integrate videos and event form

### PR DESCRIPTION
**Pull request by Boyadjie**
## 🤔 Why?

To edit the videos, they must be on the webapp and a form is required to update the data.

## 💻 How?

I've created integrated the videos in the homepage with a link to a dedicated page for each. On the dedicated page, i've build a form that send its values to the props of the player and like that it changes the content of the video.

## ✅ How to validate this PR?

- run `pnpm dev` and go to localhost:3000. You should see the home page with the two preview videos
- Click on `create video`, you should now see the video and its form under it.
- You can edit the default values with yours and it should uptade them on the video instantly

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,27 +1,86 @@
...
+					component={component}
+					inputProps={inputProps}
+					style={displayStyle}
 				/>
-				<div className={styles.thirteen}>
-					<Image priority src="/thirteen.svg" alt="13" width={40} height={31} />
-				</div>
 			</div>
+			<div className={styles.cardContent}>
+				<p>{title}</p>
```
Comment (CruuzAzul) [inline app/page.tsx:52]: `hX` or `span` ?

File: app/page.tsx [hunk 2]
```diff
@@ -1,27 +1,86 @@
...
+import {BotzEvent} from '../src/BestOfTechnozaure/Event/BotzEvent';
+import {BotzTalk} from '../src/BestOfTechnozaure/Talk/BotzTalk';
+import {EventDefaultProps} from '../src/DefaultProps/EventDefaultProps';
+import {TalkDefaultProps} from '../src/DefaultProps/TalkDefaultProps';
 
-export default function Home() {
+const displayStyle = {
+	width: 400,
+	height: 400,
+};
```
[the remark this answers is not in the document]
Comment (Boyadjie) [reply app/page.tsx:15]: Il est obligé d’être en inline celui ci sinon il n'est pas prit en compte.

File: app/[videoName]/page.tsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+'use client';
```
[the remark this answers is not in the document]
Comment (Boyadjie) [reply app/[videoName]/page.tsx:1]: Des children utilisent des useState donc il ne peut pas être enlevé

[the remark this answers is not in the document]
Comment (Boyadjie) [reply app/[videoName]/page.tsx]: j'ai passé les children en client et ça fonctionne 👍

**Review by Slocaly — APPROVED** [review]

File: app/(Components)/(Form)/BotzEvent/VideoIntro.tsx [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import {Input} from '../Input';
+import {InputDate} from '../InputDate';
+import {InputTime} from '../InputTime';
+
+export const VideoIntro: React.FC<{introInputs: object}> = ({introInputs}) => {
+	return (
+		<section>
+			{Object.entries(introInputs).map(([label, input], key) => {
+				const InputComponent =
+					label === 'date' ? InputDate : label === 'time' ? InputTime : Input;
```
Comment (CruuzAzul) [inline app/(Components)/(Form)/BotzEvent/VideoIntro.tsx:11]: Change this condition 😉 Try to avoid interweaving ternaries for better readability !

**Review by CruuzAzul — APPROVED** [review]

File: app/(Components)/(Form)/BotzEvent/VideoIntro.tsx (hunk 4 above)
Comment (Boyadjie) [reply app/(Components)/(Form)/BotzEvent/VideoIntro.tsx]: Done 👍

**Pull request merged**